scripts/package-windows-images

```diff
@@ -5,26 +5,31 @@ cd $(dirname $0)/..
 
 source ./scripts/version.sh
 
-#mkdir -p dist/artifacts
+mkdir -p dist/artifacts
 
 # 1809
-#crane --platform windows/amd64 pull ${REGISTRY}/${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-windows-amd64 rke2-windows-1809-images.tar
-#crane --platform windows/amd64 pull mcr.microsoft.com/oss/kubernetes/pause:3.5-windows-1809-amd64 rke2-windows-1809-images.tar
+crane --platform windows/amd64 pull \
+  ${REGISTRY}/${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-windows-amd64 \
+  mcr.microsoft.com/oss/kubernetes/pause:3.5-windows-1809-amd64 \
```

Comment:
as per @luthermonson we are going to use the gcr.k8s.io/pause:3.5 images as they have all the right Windows versions in them already. Long term these need to be mirrored into docker.io/rancher/mirrored-pause.

Reply:
Looks like we already have 3.5 pause images for the three versions of Windows that we are supporting (1809, 2004, 20H2) in https://hub.docker.com/r/rancher/mirrored-pause/

Is there a reason why we can't point to those pause images now?

Reply:
I don't personally care, we just need to use the same one in the airgap tarballs as the build. I'll let the two of you work it out.

Reply:
Switching back to `gcr.k8s.io/pause:3.5` after talking with Luther